src/EncryptedRow.php

```diff
@@ -482,14 +482,15 @@ public function encryptRow(
             );
             if (
                 !empty($this->aadSourceField[$field])
-                    &&
+                &&
                 \array_key_exists($this->aadSourceField[$field], $row)
             ) {
                 $aad = (string) $row[$this->aadSourceField[$field]];
             } else {
                 $aad = '';
             }
             if ($type === Constants::TYPE_JSON && !empty($this->jsonMaps[$field])) {
+                $row[$field] = $this->formatJson($row[$field]);
```

Comment:
This should probably be toggle-able. Changing this behavior for everyone would constitute a major version break,

Reply:
I will add this to the function to be an option and it's deactivated by default